fighthealthinsurance/sentry_filters.py

```
@@ -1,32 +1,172 @@
+    -- and sentry's Message interface legitimately allows
+    ``{"message": ..., "params": [...], "formatted": ...}``. Stringifying
+    handles both: the marker is still found inside a dict's repr, and nothing
+    raises. See the module docstring for why raising is not an option.
+    """
+    if isinstance(value, str):
+        return value
+    if not value:
+        return ""
+    try:
+        return str(value)
+    except Exception:  # pragma: no cover - an exotic __str__
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)